dockerfiles/evmapp/entrypoint.sh

```diff
@@ -58,6 +58,17 @@ if [ -z "${SCNODE_NET_DECLAREDADDRESS}" ]; then
 fi
 export SCNODE_NET_DECLAREDADDRESS
 
+#Custom log4j file (optional)
+LOG4J_CUSTOM_CONFIG=""
+if [ -n "${SCNODE_LOG4J_CUSTOM_CONFIG:-}" ]; then
+  if ! [ -f "${SCNODE_LOG4J_CUSTOM_CONFIG}" ]; then
+    echo "Error: Custom log4j file not found in path ${SCNODE_LOG4J_CUSTOM_CONFIG} - check env property SCNODE_LOG4J_CUSTOM_CONFIG and be sure to have the file in the container"
```

Comment:
How about the following message. Feel free to ignore :)

"Error: Custom log4j file was not found under the provided path = ${SCNODE_LOG4J_CUSTOM_CONFIG}. Please check the value of the environment variable 'SCNODE_LOG4J_CUSTOM_CONFIG' and make sure your custom log4j file is correctly mapped to the container."